lib/ui/compositing/scene_builder.cc

```diff
@@ -32,44 +32,8 @@
 
 namespace flutter {
 
-static void SceneBuilder_constructor(Dart_NativeArguments args) {
-  UIDartState::ThrowIfUIOperationsProhibited();
-  DartCallConstructor(&SceneBuilder::create, args);
-}
-
 IMPLEMENT_WRAPPERTYPEINFO(ui, SceneBuilder);
 
-#define FOR_EACH_BINDING(V)                         \
-  V(SceneBuilder, pushOffset)                       \
-  V(SceneBuilder, pushTransform)                    \
-  V(SceneBuilder, pushClipRect)                     \
-  V(SceneBuilder, pushClipRRect)                    \
-  V(SceneBuilder, pushClipPath)                     \
-  V(SceneBuilder, pushOpacity)                      \
-  V(SceneBuilder, pushColorFilter)                  \
-  V(SceneBuilder, pushImageFilter)                  \
-  V(SceneBuilder, pushBackdropFilter)               \
-  V(SceneBuilder, pushShaderMask)                   \
-  V(SceneBuilder, pushPhysicalShape)                \
-  V(SceneBuilder, pop)                              \
-  V(SceneBuilder, addPlatformView)                  \
-  V(SceneBuilder, addRetained)                      \
-  V(SceneBuilder, addPicture)                       \
-  V(SceneBuilder, addTexture)                       \
-  V(SceneBuilder, addPerformanceOverlay)            \
-  V(SceneBuilder, setRasterizerTracingThreshold)    \
-  V(SceneBuilder, setCheckerboardOffscreenLayers)   \
-  V(SceneBuilder, setCheckerboardRasterCacheImages) \
-  V(SceneBuilder, build)
-
-FOR_EACH_BINDING(DART_NATIVE_CALLBACK)
```

Comment:
Does the `DART_NATIVE_CALLBACK` macro (and possibly other macros) become dead with this change? Can they be removed?

Reply:
They're still used in the Fuchsia embedding apparently.